main.go

```diff
@@ -207,10 +207,15 @@ build:
 		} else {
 			artifacts, err = spec.artifactPaths(spec.outputPathPatterns, true)
 		}
+
 		if err != nil {
 			failf("failed to find artifacts, error: %s", err)
 		}
 
+		if len(artifacts) < 1 {
+			failf("artifact path pattern `%s` did not match any artifacts", spec.outputPathPatterns)
```

Comment:
Can you please add the project location in the error message (cfg.ProjectLocation)?
```suggestion
			failf(`Artifact path pattern (%s) did not match any artifacts on the path (%s).
Check that 'iOS/Android Output Pattern' and 'Project Location' is correct.`, spec.outputPathPatterns, spec.projectLocation)
```

Reply:
Is this something that belongs into that error message for sure? The config is printed in the log already.

Reply:
Maybe it is not clear that we are looking for the artifact under that path, recently encountered a user with this issue.
Mentioning to check both pattern and location  is enough.